addons/im_livechat/models/mail_channel.py

```diff
@@ -190,6 +191,7 @@ def _close_livechat_session(self, **kwargs):
             # Notify that the visitor has left the conversation
             self.message_post(author_id=self.env.ref('base.partner_root').id,
                               body=self._get_visitor_leave_message(**kwargs), message_type='comment', subtype_xmlid='mail.mt_comment')
+            self.env['bus.bus'].sendmany([[(self._cr.dbname, 'mail.channel', self.id), {'info': 'close_livechat_session',}]])
```

Comment:
Please add the channel id in the payload instead of relying on the "bus channel" for it.
I'm suggesting to add a second key `channel` that contains at least `id` and also maybe the new value for `livechat_active` (or maybe even the full `channel_info()` result).